source/installation.txt

```diff
@@ -20,28 +20,39 @@ For instructions on installing Relational Migrator, see the following pages:
 
      - Description
 
-     - Reference
-
    * - Local
 
-     - Suitable for evaluation and proof of concept jobs.
+     - Suitable for evaluation and proof of concept jobs. 
 
-     - :ref:`Install on a Single Machine <single-machine-install>`
+       For more information, see :ref:`Install on a Single Machine 
+       <single-machine-install>`. 
 
    * - Unattended server
 
-     - Suitable for smaller production jobs.
-
-     - :ref:`Install on an Unattended Server <unattended-server>`
+     - Suitable for smaller production jobs. 
+       
+       For more information, see :ref:`Install on an Unattended Server 
+       <unattended-server>`. 
 
    * - Kafka
 
-     - Suitable for larger production jobs. Integrates with your own Kafka 
-       infrastructure to provide enhanced resiliency.
+     - The Kafka deployment method is recommended for long-running Change Data 
+       Capture (CDC) syncs and large data migrations of more than 20GB. Apache Kafka 
+       is an open-source platform for distributed workloads that offers high 
+       availability and automatic recovery. This option is the most advanced, and is 
+       intended for users who require long-running or large production data syncs. 
+       
+       For more information, see :ref:`kafka-intro`.
+
+   * - Kafka with Confluent Cloud
+
+     - The Kafka with Confluent Cloud deployment method is an option for users who 
+       want the reliability of Kafka without having to manage their own cluster. 
+       Confluent is a fully-managed Kafka hosting provider run by the founders of 
```

Comment:
**Suggestion**

Link for additional context:

```suggestion
       `Confluent <https://www.confluent.io/>`__ is a fully-managed Kafka hosting provider run by the founders of 
```

Reply:
Updated.